doc/running_tests.md

```diff
@@ -2,44 +2,40 @@
 
-The standard behat configuration file is [behat.yml.dist](https://github.com/ezsystems/ezplatform/blob/2.5/behat.yml.dist), located in eZ Platform's main directory. There you can:
+The standard behat configuration file is [behat_ibexa_oss.yaml](https://github.com/ibexa/behat/blob/4.6/behat_ibexa_oss.yaml), located in ibexa/behat directory. There you can:
 - in the `Behat\MinkExtension` section:
   - set the URL of your website for browser testing (`base_url` parameter)
   - set driver configuration (for example `wd_host` for Selenium Server)
   - see [MinkExtension documentation](https://github.com/Behat/MinkExtension/blob/master/doc/index.rst) for more information
 - in the `Bex\Behat\ScreenshotExtension` section:
   - set your Cloudinary account details (`cloud_name` and `preset` parameter) to specify where the screenshots are uploaded
   - see [Cloudinary screenshot driver doc](https://github.com/ezsystems/behat-screenshot-image-driver-cloudinary/blob/master/README.md) for more details about the Cloudinary integration and [elvetemedve/behat-screenshot](https://github.com/elvetemedve/behat-screenshot) for other screenshots configuration options 
-- in the `Behat\Symfony2Extension` section:
-  - set Symfony environment (`env` parameter)
-  - enable Symfony debug (`debug` parameter)
-  - See [Symfony2Extension documentation](https://github.com/Behat/Symfony2Extension/blob/master/doc/index.rst) for more information
 
-Behat profiles and suites are not defined in this file, but imported from files specified at the bottom.
+Behat profiles and suites are not defined in this file, but imported from files specified at the top.
 
 ### Running browser tests
 
-If you want to run browser tests you need to have Selenium Server runnning. One way of doing this is running our Docker stack, which has built-in support for it: see [eZ Platform Docker blueprints](https://github.com/ezsystems/ezplatform/blob/master/doc/docker/README.md#behat-and-selenium-use) documentation for more information.
+If you want to run browser tests you need to have Selenium Server runnning. One way of doing this is running our Docker stack, which has built-in support for it: see [Ibexa Docker blueprints](https://github.com/ibexa/docker/blob/4.6/README.md#behat-and-selenium-use) documentation for more information.
 
-Another way is to use the Selenium Server Docker container and setting it up manually. Look at [ezplatform's .env file](https://github.com/ezsystems/ezplatform/blob/master/.env#L17) for the currently used version.
+Another way is to use the Selenium Server Docker container and setting it up manually. Look at [manifest.json file](https://github.com/ibexa/recipes-dev/blob/master/ibexa/docker/4.6/manifest.json#L23) for the currently used version.
 
 It can be set up using:
 `docker run -p 4444:4444 -p 5900:5900 --shm-size=1gb -d --name=containerName selenium/standalone-chrome-debug:3.141.59`
```

Comment:
This selenium image is 4 years old, maybe lets add here link for https://github.com/SeleniumHQ/docker-selenium 🤔